neo/Persistence/Helper.cs

```diff
@@ -63,7 +63,8 @@ public static UInt256 GetNextBlockHash(this IPersistence persistence, UInt256 ha
 
         public static long GetSysFeeAmount(this IPersistence persistence, uint height)
         {
-            return persistence.GetSysFeeAmount(Blockchain.Singleton.GetBlockHash(height));
+            var hash = Blockchain.Singleton.GetBlockHash(height);
+            return hash == null ? 0 : persistence.GetSysFeeAmount(hash);
```

Comment:
Note that this change maintain the same logic as ` GetSysFeeAmount(this IPersistence persistence, UInt256 hash)`
If the block doesn't exist, return 0

Reply:
So, what's the difference?

Reply:
In my unit tests, i haven't got a blockchain, so if i call this method with a non stored block, i get a exception without this change. There are two methods, one return 0, the other throw a exception